teku/src/test/java/tech/pegasys/teku/cli/subcommand/PeerCommandTest.java

```diff
@@ -19,19 +19,18 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import org.junit.jupiter.api.Test;
-import tech.pegasys.teku.cli.subcommand.PeerCommand.PeerGenerationParams;
 import tech.pegasys.teku.util.config.InvalidConfigurationException;
 
 public class PeerCommandTest {
 
   @Test
   public void peerGenerate_shouldDisplayErrorIfFileNotFound() {
     PeerCommand peerCommand = new PeerCommand();
-    PeerGenerationParams params = new PeerGenerationParams("directory-does-not-exist/config.dat");
+    String fileWhichDoesNotExist = "directory-does-not-exist/config.dat";
```

Comment:
nit: final, unless it needs to be overwritten later on.

Reply:
done